curriculum/challenges/chinese/05-apis-and-microservices/basic-node-and-express/serve-json-on-a-specific-route.md

```diff
@@ -1,24 +1,24 @@
 ---
 id: 587d7fb1367417b2b2512bf1
-title: Serve JSON on a Specific Route
+title: 在指定路由上提供 JSON 服务
 challengeType: 2
 forumTopicId: 301517
 dashedName: serve-json-on-a-specific-route
 ---
 
 # --description--
 
-While an HTML server serves HTML, an API serves data. A <dfn>REST</dfn> (REpresentational State Transfer) API allows data exchange in a simple way, without the need for clients to know any detail about the server. The client only needs to know where the resource is (the URL), and the action it wants to perform on it (the verb). The GET verb is used when you are fetching some information, without modifying anything. These days, the preferred data format for moving information around the web is JSON. Simply put, JSON is a convenient way to represent a JavaScript object as a string, so it can be easily transmitted.
+HTML 服务器提供 HTML 服务，而 API 提供数据服务。 <dfn>具象状态传输</dfn>（REST，REpresentational State Transfer）API 允许以简单的方式进行数据交换，对于客户端不必要知道服务器的细节。 客户只需要知道资源在哪里（URL），以及想执行的动作（动词）。 GET 动词常被用来获取无需修改的信息。 如今，网络上的移动数据首选格式是 JSON， 简而言之，JSON 是一种可以方便地用字符串表示 JavaScript 对象的方式，因此它很容易传输。
```

Comment:
Should `REST` (in the definition tags) be translated?

Reply:
@nhcarrigan It should not.  It has been fixed on Crowdin.  In an effort to get this PR merged, since I have reviewed everything else on it and think it is fine to merge, I will create an additional commit to switch it to back to `REST`.

Reply:
I am holding off on the extra commit as it will fix itself on the next download.